Makefile

```diff
@@ -848,6 +854,18 @@ keystone_kuttl: namespace input openstack_crds deploy_cleanup mariadb mariadb_de
 	make keystone_cleanup
 	make mariadb_cleanup
 
+.PHONY: cinder_kuttl_run
+cinder_kuttl_run: ## runs kuttl tests for the cinder operator, assumes that everything needed for running the test was deployed beforehand.
+	INSTALL_YAMLS=${INSTALL_YAMLS} kubectl-kuttl test --config ${CINDER_KUTTL_CONF} ${CINDER_KUTTL_DIR}
+
+.PHONY: cinder_kuttl
+cinder_kuttl: namespace input openstack_crds deploy_cleanup mariadb mariadb_deploy rabbitmq rabbitmq_deploy keystone_deploy_prep keystone keystone_deploy cinder_deploy_prep cinder infra mariadb_deploy_validate ## runs kuttl tests for the cinder operator. Installs openstack crds and cinder operators and cleans up previous deployments before running the tests and, add cleanup after running the tests.
+	make cinder_kuttl_run
+	make deploy_cleanup
```

Comment:
Some cleanup is missing here before calling `deploy_cleanup`:
```
make infra_cleanup
make rabbit_deploy_cleanup
make rabbit_clenaup
``

Reply:
thanks, adding them